draft-ietf-quic-http.md

```diff
@@ -1478,6 +1469,12 @@ SETTINGS_MAX_FRAME_SIZE:
 SETTINGS_MAX_HEADER_LIST_SIZE:
 : See {{settings-parameters}}.
 
+In HTTP/QUIC, setting values are variable-length integers (6, 14, 30, or 62 bits
+long) rather than fixed-length 32-bit fields.  This will often produce a shorter
+encoding, but will produce a longer encoding for settings which use the full
+32-bit space.  Settings ported from HTTP/2 might choose to redefine the format
+of their settings to avoid using the 62-bit encoding.
```

Comment:
I'm not sure I understand the purpose of this additional text?  You lose 2 bits on 32 bit fields, but do we really expect that will require changes on anyone's part?

Reply:
Secondary Certs currently stuffs an exporter in the setting value (to confirm both peers are on the same TLS connection with no MitM), and would overflow to an eight-byte field.  It doesn't *require* changes, but it's advisable to either take advantage of the entire eight bytes or trim two bits off to still fit in four bytes.  Hence, "might choose."

Reply:
In that case, how about starting the paragraph with 'Setting values are ...... rather than fixed length 32-bit fields as in HTTP/2."  Currently you mention HTTP/QUIC in the HTTP/QUIC draft, and don't reference the thing you're comparing against(HTTP/2).